app/src/main/java/com/github/palFinderTeam/palfinder/profile/ProfileService.kt

```diff
@@ -10,7 +10,16 @@ interface ProfileService: Repository<ProfileUser> {
     fun getLoggedInUserID(): String?
 
     /**
-     * Try to follow a user
+     * Checks if user exists in database
+     *
+     * @param userId Id of the user
+     *
+     * @return boolean
+     */
+   // suspend fun doesUserIDExist(userId: String): Boolean
```

Comment:
should we keep this fun?

Reply:
I don't really know why it was added, so I didn't delete it even though I isn't used yet